.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Summary of the discussion
+
+Describe the findings of the discussion in the issue or meeting.
+
+## Type of change (CHANGELOG.md)
+
+### Added
+- Add a new class [(#)](https://github.com/rl-institut/super-repo/pull/)
+
+### Updated
+- Update a definition [(#)](https://github.com/rl-institut/super-repo/pull/)
+
+### Removed
+- Remove a broken link [(#)](https://github.com/rl-institut/super-repo/pull/)
+
+
+## Workflow checklist
+
+### Automation
+Closes #
+
+### PR-Assignee
+- [ ] 🐙 Follow the workflow in [CONTRIBUTING.md](https://github.com/rl-institut/super-repo/blob/develop/CONTRIBUTING.md)
+- [ ] 📝 Update the [CHANGELOG.md](https://github.com/rl-institut/super-repo/blob/develop/CHANGELOG.md)
+- [ ] 📙 Update the documentation
+
+### Reviewer
+- [ ] 🐙 Follow the [Reviewer Guidelines](https://github.com/rl-institut/super-repo/blob/develop/CONTRIBUTING.md#40-let-someone-else-review-your-pr)
```

Comment:
@steull please add the "Reviewer Guidlines" to this repo & update the link or delete the line